Sources/RoktUXHelper/Utils/Bool+Extension.swift

```diff
@@ -11,7 +11,8 @@
 
 import Foundation
 
-// Compact version of SlotModel with only offer
-struct SlotOfferModel {
-    let offer: OfferModel?
+extension Bool {
```

Comment:
Can this just be replaced with another extension function in `Optional+Extension.swift`? It seems the only usage of it is in `LayoutTransformer`